tls/extensions/s2n_extension_type_lists.c

```diff
@@ -60,6 +61,7 @@ static const s2n_extension_type *const client_hello_extensions[] = {
         &s2n_client_pq_kem_extension,
         &s2n_client_renegotiation_info_extension,
         &s2n_client_cookie_extension,
+        &s2n_signature_algorithms_cert_extension,
```

Comment:
What about client auth? My understanding of this extension is that the server can also send it with the CertificateRequest to affect the signature algorithm used for client auth.

Reply:
Yes, but we don't actually take the server's preference into account when we select a cert for client auth.